library/CHART-TEMPLATE/tests/common_library_labels_test.yaml

```
@@ -0,0 +1,43 @@
+suite: test labels helpers
+templates:
+  - templates/deployment.yaml # we could check the same for service and serviceAccount as well
+release:
+  name: my-release
+  namespace: my-namespace
+set:
+  licenseKey: test-license-key
+  cluster: test-cluster
+tests:
+  - it: sets labels according to values
+    set:
+      labels:
+        helm.sh/chart: fixed
+    asserts:
+      - equal:
```

what about using [contains](https://github.com/vbehar/helm3-unittest/blob/master/unittest/assertion.go#L125) . So we just check what the test says. ( I never tried this `contains` so it might be not the thing we want)
```suggestion
      - contains:
           path: metadata.labels
           value:
             helm.sh/chart: fixed
```

---

this would work!

---

actually no, contains does not work as I expected 😖

---

As the documentation is almost inexistent I use this example as a guide: https://github.com/quintush/helm-unittest/blob/master/test/data/v3/basic/tests_failed/deployment_test.yaml

`contains` is for arrays.